modules/ocf/manifests/ssl/lets_encrypt/dns.pp

```diff
@@ -5,6 +5,10 @@
 ) {
   require ocf::ssl::lets_encrypt::dns_common
 
+  # we use this package in the le_cert_info fact, to gather information about
+  # whatever certs are currently present
+  package { 'python3-cryptography':; }
```

Comment:
you may also want to consider adding it to the preseed file (https://github.com/ocf/puppet/blob/master/modules/ocf_dhcp/files/preseed/buster#L91) since facts will be collected before any manifests run on new hosts

Reply:
I thought about this but I don't think it's necessary because the fact won't crash on new hosts (since the import is guarded) and the fact will be empty on new hosts anyway (since they won't have certs yet).